src/components/policy/test/policy_manager_impl_test.cc

```diff
@@ -574,6 +574,104 @@ TEST_F(PolicyManagerImplTest, LoadPT_SetInvalidUpdatePT_PTIsNotLoaded) {
   EXPECT_FALSE(manager->LoadPT("file_pt_update.json", msg));
 }
 
+// [GENIVI] Policies: "device" and "pre_DataConsent" sections
+// must be present in PT
+TEST_F(PolicyManagerImplTest,
+       CRQ26921_LoadPT_SetInvalidUpdatePT_PTIsNotLoaded) {
```

Comment:
@orhan-mehmedov Bad unit-test case name. From the name of UT it should be clear what it tests. Nobody can understand what does this `CRQ26921` mean